utils/compression/type.go

```diff
@@ -14,16 +14,14 @@ type Type byte
 
 const (
 	TypeNone Type = iota + 1
-	TypeGzip      // Remove once v1.11.x is out.
```

Comment:
I don't think this value is ever sent over the wire so I _think_ it's safe to just delete instead of marking as a placeholder in this enum...